XamarinCommunityToolkit/Converters/StringIsNotNullOrEmptyConverter.shared.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Converters
+{
+    public class StringIsNotNullOrEmptyConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture) => !string.IsNullOrEmpty(value?.ToString());
```

Comment:
Perhaps, it makes sense to cast the value to "string" explicitly, because it another case that's possible to use this converted with any type and you won't get an exception.